specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
```

Comment:
With this resource type definition, I believe you also need to support 'point get', i.e. get a single usage by its name? At least, I hope to clarify this. Potential [ARMMustGuidelineViolation] /cc @rkmanda

Reply:
I spoke with Zhenhua about this previously. This an API primarily for onboarding to QMS, and other RPs have previously implemented it, and did not have to create a point get method as well. Here is an example from Microsoft.LabServices: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/labservices/resource-manager/Microsoft.LabServices/stable/2023-06-07/Usages.json

Reply:
@rohinisharma OK, I hope this is just a special case then. Its not an internal API right?

Reply:
Correct, this is not an internal API.